src/adapters/communities-db.ts

```diff
@@ -337,6 +335,79 @@ export function createCommunitiesDBComponent(
         DELETE FROM community_members WHERE community_id = ${communityId} AND member_address = ${normalizeAddress(memberAddress)}
       `
       await pg.query(query)
+    },
+
+    async banMemberFromCommunity(
```

Comment:
I've iterated over this approach with a teammate after we agreed on keeping the banned members in the list of members of a community and we realized the following:
- If we remove the banned members from the communities' list members we don't need to perform JOINs while returning communities' members
- I re-call I proposed keeping them in the list of members for tracking capabilities but since we have a specific table stating all the banned members I think that's no longer needed

What do you think?